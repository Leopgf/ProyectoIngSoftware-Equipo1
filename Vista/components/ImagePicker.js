import React, { useState, useEffect } from 'react';
import { Image, View, Platform, Dimensions } from 'react-native';
import * as ImagePicker from 'expo-image-picker';
import { StyleSheet } from 'react-native';
import { Button } from 'galio-framework';
import { cloudinary } from 'cloudinary-react';

export default function ImagePickerExample({ onImagePicked, defaultImage }) {
  const { width, height } = Dimensions.get('screen');
  const [image, setImage] = useState(null);

  useEffect(() => {
    (async () => {
      if (Platform.OS !== 'web') {
        const { status } = await ImagePicker.requestCameraRollPermissionsAsync();
        if (status !== 'granted') {
          alert('Sorry, we need camera roll permissions to make this work!');
        }
      }
    })();
  }, []);

  const pickImage = async () => {
    let result = await ImagePicker.launchImageLibraryAsync({
      mediaTypes: ImagePicker.MediaTypeOptions.All,
      allowsEditing: true,
      aspect: [4, 3],
      quality: 1,
    });

    if (!result.cancelled) {
      let file = {
        uri: result.uri,
        type: `test/${result.uri.split('.').pop()}`,
        name: result.uri,
      };

      let apiUrl = 'https://api.cloudinary.com/v1_1/mixosteam/image/upload';

      const formData = new FormData();
      formData.append('file', file);
      formData.append('upload_preset', 'zc9ttas7');
      formData.append('cloud_name', 'mixosteam');

      await fetch(apiUrl, {
        body: formData,
        method: 'POST',
      })
        .then(async (r) => await r.json())
        .then((dato) => {
          console.log('TOY AQUIIII');
          setImage(dato.secure_url);
          onImagePicked({ uri: dato.secure_url });
        })
        .catch((error) => {
          console.log(error);
        });
    }
  };

  const styles = StyleSheet.create({
    smallButton: {
      width: 75,
      height: 28,
    },
    Button: {
      width: width / 2.5,
      height: height / 20,
      shadowOpacity: 0,
    },
  });

  return (
    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center',shadowOpacity: 0}}>
      <Button style={styles.Button} onPress={pickImage}>
        Subir imagen
      </Button>
      {image && (
        <Image source={{ uri: image }} style={{ width: 200, height: 200, marginTop: 20 }} />
      )}
      {defaultImage && !image && (
        <Image source={{ uri: defaultImage }} style={{ width: 200, height: 200, marginTop: 20 }} />
      )}
    </View>
  );
}
