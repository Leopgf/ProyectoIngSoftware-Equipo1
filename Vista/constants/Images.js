// local imgs
const Onboarding = require('../assets/imgs/FFondo.jpg');
const Logo = require('../assets/imgs/logop.png');
const ProfileBackground = require('../assets/imgs/basePerfil.jpg');
const RegisterBackground = require('../assets/imgs/fondoInicio.jpg');
const Pro = require('../assets/imgs/bg.png');
const NowLogo = require('../assets/imgs/logo.png');
const iOSLogo = require('../assets/imgs/ios.png');
const androidLogo = require('../assets/imgs/android.png');
const ProfilePicture = require('../assets/imgs/profile-img.jpg');
const Conversor = require('../assets/imgs/conversor.jpg');
const AddReceta = require('../assets/imgs/addReceta.jpg');
const CreativeTimLogo = require('../assets/imgs/creative-tim-white-slim2.png');
const InvisionLogo = require('../assets/imgs/invision-white-slim.png');

const Viewed = [
  require('../assets/imgs/project5.jpg'),
  require('../assets/imgs/project7.jpg'),
  require('../assets/imgs/project6.jpg'),
  require('../assets/imgs/project4.jpg'),
  require('../assets/imgs/project21.jpg'),
  require('../assets/imgs/project24.jpg')
];

const Products = {
  text: 'View article',
  path: require('../assets/imgs/bg15.jpg')
};

export default {
  Onboarding,
  Logo,
  ProfileBackground,
  Conversor,
  ProfilePicture,
  RegisterBackground,
  AddReceta,
  Viewed,
  Pro,
  Products,
  NowLogo: NowLogo
};
