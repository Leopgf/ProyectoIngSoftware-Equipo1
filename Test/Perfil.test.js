import React from 'react';
import renderer from 'react-test-renderer';

import { StyleSheet, Dimensions, ImageBackground } from 'react-native';
import { Block, Text, theme, Button as GaButton } from 'galio-framework';
import { Card } from '../Vista/components';
import { Images, nowTheme } from '../Vista/constants';

const { width, height } = Dimensions.get('screen');

const thumbMeasure = (width - 48 - 32) / 3;

describe('<Perfil />', () => {
    //
    it('renders correctly', () => {
        const tree = renderer.create(
            <ImageBackground
              source={Images.ProfileBackground}
              style={styles.profileContainer}
              imageStyle={styles.profileBackground}
            >
              <Block flex style={styles.profileCard}>
                <Block
                  style={{ position: 'absolute', width: width, zIndex: 5, paddingHorizontal: 20 }}
                >
                  <Block style={{ top: height * 0.23 }}>
                    <Block middle>
                      {/*nombre usuario*/}
                      <Text
                        style={{
                          fontFamily: 'montserrat-bold',
                          marginBottom: theme.SIZES.BASE / 2,
                          fontWeight: '900',
                          fontSize: 30,
                        }}
                        color="#e63746"
                      >
                        Nombre Apellido
                      </Text>
                      {/*texto usuario de MixoÂ´s*/}
                      <Text
                        size={16}
                        color="#0f1e2e"
                        style={{ marginTop: 5, lineHeight: 20, fontSize: 18, opacity: 0.8 }}
                      >
                        @Usuario
                      </Text>
                    </Block>
                  </Block>
                </Block>
              </Block>
            </ImageBackground>
        ).toJSON();
        expect(tree).toMatchSnapshot();
      });
    it('renders correctly', () => {
        const tree = renderer.create(
            <Text
                  style={{
                    fontFamily: 'montserrat-bold',
                    marginBottom: theme.SIZES.BASE / 2,
                    fontWeight: '900',
                    fontSize: 25,
                  }}
                  color="#e63746"
                >
                  Mi Biblioteca
                </Text>
        ).toJSON();
        expect(tree).toMatchSnapshot();
      });
    it('renders correctly', () => {
        const tree = renderer.create(
            <Block flex>
                  <Block flex row>
                  </Block>
              </Block>
        ).toJSON();
        expect(tree).toMatchSnapshot();
      });
});

const styles = StyleSheet.create({
    profileContainer: {
      width,
      height,
      padding: 0,
      zIndex: 1,
    },
    profileBackground: {
      width,
      height: height * 0.35,
    },
  
    info: {
      marginTop: 30,
      paddingHorizontal: 10,
      height: height * 0.8,
    },
    avatarContainer: {
      position: 'relative',
      marginTop: -80,
    },
    avatar: {
      width: thumbMeasure,
      height: thumbMeasure,
      borderRadius: 50,
      borderWidth: 0,
    },
    nameInfo: {
      marginTop: 35,
    },
    thumb: {
      borderRadius: 4,
      marginVertical: 4,
      alignSelf: 'center',
      width: thumbMeasure,
      height: thumbMeasure,
    },
    social: {
      width: nowTheme.SIZES.BASE * 3,
      height: nowTheme.SIZES.BASE * 3,
      borderRadius: nowTheme.SIZES.BASE * 1.5,
      justifyContent: 'center',
      zIndex: 99,
      marginHorizontal: 5,
    },
  });  
  
