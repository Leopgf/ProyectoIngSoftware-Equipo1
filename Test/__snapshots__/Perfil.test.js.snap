// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Perfil /> renders correctly 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "height": 1334,
      "padding": 0,
      "width": 750,
      "zIndex": 1,
    }
  }
>
  <Image
    source={1}
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": 1334,
          "width": 750,
        },
        Object {
          "height": 466.9,
          "width": 750,
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "flexDirection": "column",
        },
        false,
        Object {
          "flex": 1,
        },
        false,
        false,
        false,
        false,
        false,
        false,
        null,
        false,
        false,
        false,
        null,
        null,
        null,
        undefined,
      ]
    }
    theme={
      Object {
        "COLORS": Object {
          "BLACK": "#000000",
          "BLOCK": "#808080",
          "DRIBBBLE": "#EA4C89",
          "ERROR": "#FE2472",
          "FACEBOOK": "#3B5998",
          "GREY": "#898989",
          "ICON": "#000000",
          "INFO": "#1232FF",
          "INPUT": "#808080",
          "MUTED": "#9FA5AA",
          "NAVBAR": "#F9F9F9",
          "NEUTRAL": "rgba(255,255,255, 0.65)",
          "PLACEHOLDER": "#9FA5AA",
          "PRIMARY": "#B23AFC",
          "SUCCESS": "#45DF31",
          "THEME": "#B23AFC",
          "TRANSPARENT": "transparent",
          "TWITTER": "#5BC0DE",
          "WARNING": "#FF9C09",
          "WHITE": "#FFFFFF",
        },
        "SIZES": Object {
          "ANDROID_ELEVATION": 1,
          "BASE": 16,
          "BLOCK_SHADOW_OPACITY": 0.15,
          "BLOCK_SHADOW_RADIUS": 8,
          "BORDER_RADIUS": 6,
          "BORDER_WIDTH": 0.8,
          "BUTTON_HEIGHT": 44,
          "BUTTON_SHADOW_RADIUS": 10,
          "BUTTON_WIDTH": 144,
          "CARD_AVATAR_HEIGHT": 40,
          "CARD_AVATAR_RADIUS": 20,
          "CARD_AVATAR_WIDTH": 40,
          "CARD_BORDER_RADIUS": 6.4,
          "CARD_BORDER_WIDTH": 0.8,
          "CARD_FOOTER_HORIZONTAL": 12,
          "CARD_FOOTER_VERTICAL": 12,
          "CARD_IMAGE_HEIGHT": 200,
          "CARD_MARGIN_VERTICAL": 14,
          "CARD_ROUND": 3,
          "CARD_ROUNDED": 8,
          "CARD_WIDTH": 718,
          "CHECKBOX_HEIGHT": 20,
          "CHECKBOX_WIDTH": 20,
          "FONT": 16,
          "ICON": 16,
          "INPUT_BORDER_RADIUS": 8,
          "INPUT_BORDER_WIDTH": 0.8,
          "INPUT_HEIGHT": 44,
          "INPUT_HELP_TEXT": 12.8,
          "INPUT_HORIZONTAL": 16,
          "INPUT_LABEL_BOTTOM": 4,
          "INPUT_LABEL_TEXT": 14.4,
          "INPUT_ROUNDED": 27.2,
          "INPUT_TEXT": 14,
          "NAVBAR_HEIGHT": 66,
          "NAVBAR_LEFT_FLEX": 0.5,
          "NAVBAR_LEFT_HEIGHT": 93.38000000000001,
          "NAVBAR_LEFT_MARGIN": 16,
          "NAVBAR_RIGHT_FLEX": 0.5,
          "NAVBAR_RIGHT_HEIGHT": 93.38000000000001,
          "NAVBAR_RIGHT_MARGIN": 16,
          "NAVBAR_TITLE_FLEX": 2,
          "NAVBAR_TITLE_HEIGHT": 93.38000000000001,
          "NAVBAR_TITLE_TEXT": 14,
          "NAVBAR_VERTICAL": 16,
          "OPACITY": 0.8,
          "RADIO_HEIGHT": 24,
          "RADIO_THICKNESS": 2,
          "RADIO_WIDTH": 24,
          "THUMB_SIZE": 25,
          "TRACK_SIZE": 4,
        },
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "flexDirection": "column",
          },
          false,
          false,
          false,
          false,
          false,
          false,
          false,
          false,
          null,
          false,
          false,
          false,
          null,
          null,
          null,
          Object {
            "paddingHorizontal": 20,
            "position": "absolute",
            "width": 750,
            "zIndex": 5,
          },
        ]
      }
      theme={
        Object {
          "COLORS": Object {
            "BLACK": "#000000",
            "BLOCK": "#808080",
            "DRIBBBLE": "#EA4C89",
            "ERROR": "#FE2472",
            "FACEBOOK": "#3B5998",
            "GREY": "#898989",
            "ICON": "#000000",
            "INFO": "#1232FF",
            "INPUT": "#808080",
            "MUTED": "#9FA5AA",
            "NAVBAR": "#F9F9F9",
            "NEUTRAL": "rgba(255,255,255, 0.65)",
            "PLACEHOLDER": "#9FA5AA",
            "PRIMARY": "#B23AFC",
            "SUCCESS": "#45DF31",
            "THEME": "#B23AFC",
            "TRANSPARENT": "transparent",
            "TWITTER": "#5BC0DE",
            "WARNING": "#FF9C09",
            "WHITE": "#FFFFFF",
          },
          "SIZES": Object {
            "ANDROID_ELEVATION": 1,
            "BASE": 16,
            "BLOCK_SHADOW_OPACITY": 0.15,
            "BLOCK_SHADOW_RADIUS": 8,
            "BORDER_RADIUS": 6,
            "BORDER_WIDTH": 0.8,
            "BUTTON_HEIGHT": 44,
            "BUTTON_SHADOW_RADIUS": 10,
            "BUTTON_WIDTH": 144,
            "CARD_AVATAR_HEIGHT": 40,
            "CARD_AVATAR_RADIUS": 20,
            "CARD_AVATAR_WIDTH": 40,
            "CARD_BORDER_RADIUS": 6.4,
            "CARD_BORDER_WIDTH": 0.8,
            "CARD_FOOTER_HORIZONTAL": 12,
            "CARD_FOOTER_VERTICAL": 12,
            "CARD_IMAGE_HEIGHT": 200,
            "CARD_MARGIN_VERTICAL": 14,
            "CARD_ROUND": 3,
            "CARD_ROUNDED": 8,
            "CARD_WIDTH": 718,
            "CHECKBOX_HEIGHT": 20,
            "CHECKBOX_WIDTH": 20,
            "FONT": 16,
            "ICON": 16,
            "INPUT_BORDER_RADIUS": 8,
            "INPUT_BORDER_WIDTH": 0.8,
            "INPUT_HEIGHT": 44,
            "INPUT_HELP_TEXT": 12.8,
            "INPUT_HORIZONTAL": 16,
            "INPUT_LABEL_BOTTOM": 4,
            "INPUT_LABEL_TEXT": 14.4,
            "INPUT_ROUNDED": 27.2,
            "INPUT_TEXT": 14,
            "NAVBAR_HEIGHT": 66,
            "NAVBAR_LEFT_FLEX": 0.5,
            "NAVBAR_LEFT_HEIGHT": 93.38000000000001,
            "NAVBAR_LEFT_MARGIN": 16,
            "NAVBAR_RIGHT_FLEX": 0.5,
            "NAVBAR_RIGHT_HEIGHT": 93.38000000000001,
            "NAVBAR_RIGHT_MARGIN": 16,
            "NAVBAR_TITLE_FLEX": 2,
            "NAVBAR_TITLE_HEIGHT": 93.38000000000001,
            "NAVBAR_TITLE_TEXT": 14,
            "NAVBAR_VERTICAL": 16,
            "OPACITY": 0.8,
            "RADIO_HEIGHT": 24,
            "RADIO_THICKNESS": 2,
            "RADIO_WIDTH": 24,
            "THUMB_SIZE": 25,
            "TRACK_SIZE": 4,
          },
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "flexDirection": "column",
            },
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            false,
            null,
            false,
            false,
            false,
            null,
            null,
            null,
            Object {
              "top": 306.82,
            },
          ]
        }
        theme={
          Object {
            "COLORS": Object {
              "BLACK": "#000000",
              "BLOCK": "#808080",
              "DRIBBBLE": "#EA4C89",
              "ERROR": "#FE2472",
              "FACEBOOK": "#3B5998",
              "GREY": "#898989",
              "ICON": "#000000",
              "INFO": "#1232FF",
              "INPUT": "#808080",
              "MUTED": "#9FA5AA",
              "NAVBAR": "#F9F9F9",
              "NEUTRAL": "rgba(255,255,255, 0.65)",
              "PLACEHOLDER": "#9FA5AA",
              "PRIMARY": "#B23AFC",
              "SUCCESS": "#45DF31",
              "THEME": "#B23AFC",
              "TRANSPARENT": "transparent",
              "TWITTER": "#5BC0DE",
              "WARNING": "#FF9C09",
              "WHITE": "#FFFFFF",
            },
            "SIZES": Object {
              "ANDROID_ELEVATION": 1,
              "BASE": 16,
              "BLOCK_SHADOW_OPACITY": 0.15,
              "BLOCK_SHADOW_RADIUS": 8,
              "BORDER_RADIUS": 6,
              "BORDER_WIDTH": 0.8,
              "BUTTON_HEIGHT": 44,
              "BUTTON_SHADOW_RADIUS": 10,
              "BUTTON_WIDTH": 144,
              "CARD_AVATAR_HEIGHT": 40,
              "CARD_AVATAR_RADIUS": 20,
              "CARD_AVATAR_WIDTH": 40,
              "CARD_BORDER_RADIUS": 6.4,
              "CARD_BORDER_WIDTH": 0.8,
              "CARD_FOOTER_HORIZONTAL": 12,
              "CARD_FOOTER_VERTICAL": 12,
              "CARD_IMAGE_HEIGHT": 200,
              "CARD_MARGIN_VERTICAL": 14,
              "CARD_ROUND": 3,
              "CARD_ROUNDED": 8,
              "CARD_WIDTH": 718,
              "CHECKBOX_HEIGHT": 20,
              "CHECKBOX_WIDTH": 20,
              "FONT": 16,
              "ICON": 16,
              "INPUT_BORDER_RADIUS": 8,
              "INPUT_BORDER_WIDTH": 0.8,
              "INPUT_HEIGHT": 44,
              "INPUT_HELP_TEXT": 12.8,
              "INPUT_HORIZONTAL": 16,
              "INPUT_LABEL_BOTTOM": 4,
              "INPUT_LABEL_TEXT": 14.4,
              "INPUT_ROUNDED": 27.2,
              "INPUT_TEXT": 14,
              "NAVBAR_HEIGHT": 66,
              "NAVBAR_LEFT_FLEX": 0.5,
              "NAVBAR_LEFT_HEIGHT": 93.38000000000001,
              "NAVBAR_LEFT_MARGIN": 16,
              "NAVBAR_RIGHT_FLEX": 0.5,
              "NAVBAR_RIGHT_HEIGHT": 93.38000000000001,
              "NAVBAR_RIGHT_MARGIN": 16,
              "NAVBAR_TITLE_FLEX": 2,
              "NAVBAR_TITLE_HEIGHT": 93.38000000000001,
              "NAVBAR_TITLE_TEXT": 14,
              "NAVBAR_VERTICAL": 16,
              "OPACITY": 0.8,
              "RADIO_HEIGHT": 24,
              "RADIO_THICKNESS": 2,
              "RADIO_WIDTH": 24,
              "THUMB_SIZE": 25,
              "TRACK_SIZE": 4,
            },
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "flexDirection": "column",
              },
              false,
              false,
              false,
              Object {
                "alignItems": "center",
                "justifyContent": "center",
              },
              false,
              false,
              false,
              false,
              null,
              false,
              false,
              false,
              null,
              null,
              null,
              undefined,
            ]
          }
          theme={
            Object {
              "COLORS": Object {
                "BLACK": "#000000",
                "BLOCK": "#808080",
                "DRIBBBLE": "#EA4C89",
                "ERROR": "#FE2472",
                "FACEBOOK": "#3B5998",
                "GREY": "#898989",
                "ICON": "#000000",
                "INFO": "#1232FF",
                "INPUT": "#808080",
                "MUTED": "#9FA5AA",
                "NAVBAR": "#F9F9F9",
                "NEUTRAL": "rgba(255,255,255, 0.65)",
                "PLACEHOLDER": "#9FA5AA",
                "PRIMARY": "#B23AFC",
                "SUCCESS": "#45DF31",
                "THEME": "#B23AFC",
                "TRANSPARENT": "transparent",
                "TWITTER": "#5BC0DE",
                "WARNING": "#FF9C09",
                "WHITE": "#FFFFFF",
              },
              "SIZES": Object {
                "ANDROID_ELEVATION": 1,
                "BASE": 16,
                "BLOCK_SHADOW_OPACITY": 0.15,
                "BLOCK_SHADOW_RADIUS": 8,
                "BORDER_RADIUS": 6,
                "BORDER_WIDTH": 0.8,
                "BUTTON_HEIGHT": 44,
                "BUTTON_SHADOW_RADIUS": 10,
                "BUTTON_WIDTH": 144,
                "CARD_AVATAR_HEIGHT": 40,
                "CARD_AVATAR_RADIUS": 20,
                "CARD_AVATAR_WIDTH": 40,
                "CARD_BORDER_RADIUS": 6.4,
                "CARD_BORDER_WIDTH": 0.8,
                "CARD_FOOTER_HORIZONTAL": 12,
                "CARD_FOOTER_VERTICAL": 12,
                "CARD_IMAGE_HEIGHT": 200,
                "CARD_MARGIN_VERTICAL": 14,
                "CARD_ROUND": 3,
                "CARD_ROUNDED": 8,
                "CARD_WIDTH": 718,
                "CHECKBOX_HEIGHT": 20,
                "CHECKBOX_WIDTH": 20,
                "FONT": 16,
                "ICON": 16,
                "INPUT_BORDER_RADIUS": 8,
                "INPUT_BORDER_WIDTH": 0.8,
                "INPUT_HEIGHT": 44,
                "INPUT_HELP_TEXT": 12.8,
                "INPUT_HORIZONTAL": 16,
                "INPUT_LABEL_BOTTOM": 4,
                "INPUT_LABEL_TEXT": 14.4,
                "INPUT_ROUNDED": 27.2,
                "INPUT_TEXT": 14,
                "NAVBAR_HEIGHT": 66,
                "NAVBAR_LEFT_FLEX": 0.5,
                "NAVBAR_LEFT_HEIGHT": 93.38000000000001,
                "NAVBAR_LEFT_MARGIN": 16,
                "NAVBAR_RIGHT_FLEX": 0.5,
                "NAVBAR_RIGHT_HEIGHT": 93.38000000000001,
                "NAVBAR_RIGHT_MARGIN": 16,
                "NAVBAR_TITLE_FLEX": 2,
                "NAVBAR_TITLE_HEIGHT": 93.38000000000001,
                "NAVBAR_TITLE_TEXT": 14,
                "NAVBAR_VERTICAL": 16,
                "OPACITY": 0.8,
                "RADIO_HEIGHT": 24,
                "RADIO_THICKNESS": 2,
                "RADIO_WIDTH": 24,
                "THUMB_SIZE": 25,
                "TRACK_SIZE": 4,
              },
            }
          }
        >
          <Text
            style={
              Array [
                false,
                false,
                false,
                false,
                false,
                false,
                false,
                undefined,
                0,
                Object {
                  "color": "#e63746",
                },
                false,
                false,
                undefined,
                Object {
                  "fontFamily": "montserrat-bold",
                  "fontSize": 30,
                  "fontWeight": "900",
                  "marginBottom": 8,
                },
              ]
            }
          >
            Nombre Apellido
          </Text>
          <Text
            style={
              Array [
                false,
                false,
                false,
                false,
                false,
                false,
                false,
                undefined,
                Object {
                  "fontSize": 16,
                },
                Object {
                  "color": "#0f1e2e",
                },
                false,
                false,
                undefined,
                Object {
                  "fontSize": 18,
                  "lineHeight": 20,
                  "marginTop": 5,
                  "opacity": 0.8,
                },
              ]
            }
          >
            @Usuario
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`<Perfil /> renders correctly 2`] = `
<Text
  style={
    Array [
      false,
      false,
      false,
      false,
      false,
      false,
      false,
      undefined,
      0,
      Object {
        "color": "#e63746",
      },
      false,
      false,
      undefined,
      Object {
        "fontFamily": "montserrat-bold",
        "fontSize": 25,
        "fontWeight": "900",
        "marginBottom": 8,
      },
    ]
  }
>
  Mi Biblioteca
</Text>
`;

exports[`<Perfil /> renders correctly 3`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "column",
      },
      false,
      Object {
        "flex": 1,
      },
      false,
      false,
      false,
      false,
      false,
      false,
      null,
      false,
      false,
      false,
      null,
      null,
      null,
      undefined,
    ]
  }
  theme={
    Object {
      "COLORS": Object {
        "BLACK": "#000000",
        "BLOCK": "#808080",
        "DRIBBBLE": "#EA4C89",
        "ERROR": "#FE2472",
        "FACEBOOK": "#3B5998",
        "GREY": "#898989",
        "ICON": "#000000",
        "INFO": "#1232FF",
        "INPUT": "#808080",
        "MUTED": "#9FA5AA",
        "NAVBAR": "#F9F9F9",
        "NEUTRAL": "rgba(255,255,255, 0.65)",
        "PLACEHOLDER": "#9FA5AA",
        "PRIMARY": "#B23AFC",
        "SUCCESS": "#45DF31",
        "THEME": "#B23AFC",
        "TRANSPARENT": "transparent",
        "TWITTER": "#5BC0DE",
        "WARNING": "#FF9C09",
        "WHITE": "#FFFFFF",
      },
      "SIZES": Object {
        "ANDROID_ELEVATION": 1,
        "BASE": 16,
        "BLOCK_SHADOW_OPACITY": 0.15,
        "BLOCK_SHADOW_RADIUS": 8,
        "BORDER_RADIUS": 6,
        "BORDER_WIDTH": 0.8,
        "BUTTON_HEIGHT": 44,
        "BUTTON_SHADOW_RADIUS": 10,
        "BUTTON_WIDTH": 144,
        "CARD_AVATAR_HEIGHT": 40,
        "CARD_AVATAR_RADIUS": 20,
        "CARD_AVATAR_WIDTH": 40,
        "CARD_BORDER_RADIUS": 6.4,
        "CARD_BORDER_WIDTH": 0.8,
        "CARD_FOOTER_HORIZONTAL": 12,
        "CARD_FOOTER_VERTICAL": 12,
        "CARD_IMAGE_HEIGHT": 200,
        "CARD_MARGIN_VERTICAL": 14,
        "CARD_ROUND": 3,
        "CARD_ROUNDED": 8,
        "CARD_WIDTH": 718,
        "CHECKBOX_HEIGHT": 20,
        "CHECKBOX_WIDTH": 20,
        "FONT": 16,
        "ICON": 16,
        "INPUT_BORDER_RADIUS": 8,
        "INPUT_BORDER_WIDTH": 0.8,
        "INPUT_HEIGHT": 44,
        "INPUT_HELP_TEXT": 12.8,
        "INPUT_HORIZONTAL": 16,
        "INPUT_LABEL_BOTTOM": 4,
        "INPUT_LABEL_TEXT": 14.4,
        "INPUT_ROUNDED": 27.2,
        "INPUT_TEXT": 14,
        "NAVBAR_HEIGHT": 66,
        "NAVBAR_LEFT_FLEX": 0.5,
        "NAVBAR_LEFT_HEIGHT": 93.38000000000001,
        "NAVBAR_LEFT_MARGIN": 16,
        "NAVBAR_RIGHT_FLEX": 0.5,
        "NAVBAR_RIGHT_HEIGHT": 93.38000000000001,
        "NAVBAR_RIGHT_MARGIN": 16,
        "NAVBAR_TITLE_FLEX": 2,
        "NAVBAR_TITLE_HEIGHT": 93.38000000000001,
        "NAVBAR_TITLE_TEXT": 14,
        "NAVBAR_VERTICAL": 16,
        "OPACITY": 0.8,
        "RADIO_HEIGHT": 24,
        "RADIO_THICKNESS": 2,
        "RADIO_WIDTH": 24,
        "THUMB_SIZE": 25,
        "TRACK_SIZE": 4,
      },
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flexDirection": "column",
        },
        Object {
          "flexDirection": "row",
        },
        Object {
          "flex": 1,
        },
        false,
        false,
        false,
        false,
        false,
        false,
        null,
        false,
        false,
        false,
        null,
        null,
        null,
        undefined,
      ]
    }
    theme={
      Object {
        "COLORS": Object {
          "BLACK": "#000000",
          "BLOCK": "#808080",
          "DRIBBBLE": "#EA4C89",
          "ERROR": "#FE2472",
          "FACEBOOK": "#3B5998",
          "GREY": "#898989",
          "ICON": "#000000",
          "INFO": "#1232FF",
          "INPUT": "#808080",
          "MUTED": "#9FA5AA",
          "NAVBAR": "#F9F9F9",
          "NEUTRAL": "rgba(255,255,255, 0.65)",
          "PLACEHOLDER": "#9FA5AA",
          "PRIMARY": "#B23AFC",
          "SUCCESS": "#45DF31",
          "THEME": "#B23AFC",
          "TRANSPARENT": "transparent",
          "TWITTER": "#5BC0DE",
          "WARNING": "#FF9C09",
          "WHITE": "#FFFFFF",
        },
        "SIZES": Object {
          "ANDROID_ELEVATION": 1,
          "BASE": 16,
          "BLOCK_SHADOW_OPACITY": 0.15,
          "BLOCK_SHADOW_RADIUS": 8,
          "BORDER_RADIUS": 6,
          "BORDER_WIDTH": 0.8,
          "BUTTON_HEIGHT": 44,
          "BUTTON_SHADOW_RADIUS": 10,
          "BUTTON_WIDTH": 144,
          "CARD_AVATAR_HEIGHT": 40,
          "CARD_AVATAR_RADIUS": 20,
          "CARD_AVATAR_WIDTH": 40,
          "CARD_BORDER_RADIUS": 6.4,
          "CARD_BORDER_WIDTH": 0.8,
          "CARD_FOOTER_HORIZONTAL": 12,
          "CARD_FOOTER_VERTICAL": 12,
          "CARD_IMAGE_HEIGHT": 200,
          "CARD_MARGIN_VERTICAL": 14,
          "CARD_ROUND": 3,
          "CARD_ROUNDED": 8,
          "CARD_WIDTH": 718,
          "CHECKBOX_HEIGHT": 20,
          "CHECKBOX_WIDTH": 20,
          "FONT": 16,
          "ICON": 16,
          "INPUT_BORDER_RADIUS": 8,
          "INPUT_BORDER_WIDTH": 0.8,
          "INPUT_HEIGHT": 44,
          "INPUT_HELP_TEXT": 12.8,
          "INPUT_HORIZONTAL": 16,
          "INPUT_LABEL_BOTTOM": 4,
          "INPUT_LABEL_TEXT": 14.4,
          "INPUT_ROUNDED": 27.2,
          "INPUT_TEXT": 14,
          "NAVBAR_HEIGHT": 66,
          "NAVBAR_LEFT_FLEX": 0.5,
          "NAVBAR_LEFT_HEIGHT": 93.38000000000001,
          "NAVBAR_LEFT_MARGIN": 16,
          "NAVBAR_RIGHT_FLEX": 0.5,
          "NAVBAR_RIGHT_HEIGHT": 93.38000000000001,
          "NAVBAR_RIGHT_MARGIN": 16,
          "NAVBAR_TITLE_FLEX": 2,
          "NAVBAR_TITLE_HEIGHT": 93.38000000000001,
          "NAVBAR_TITLE_TEXT": 14,
          "NAVBAR_VERTICAL": 16,
          "OPACITY": 0.8,
          "RADIO_HEIGHT": 24,
          "RADIO_THICKNESS": 2,
          "RADIO_WIDTH": 24,
          "THUMB_SIZE": 25,
          "TRACK_SIZE": 4,
        },
      }
    }
  />
</View>
`;
